server/app/routes/v1/nodes_api.rb

```diff
@@ -82,6 +81,21 @@ def load_grid_node(grid_name, node_id)
           end
         end
 
+        r.post do
+          # POST /v1/nodes/:grid/:node/availability
+          r.on 'availability' do
```

Comment:
yes, as per the internal discussion functionality was incorporated into node update API and forgot to remove these